app/models/program.rb

```diff
@@ -14,6 +14,7 @@ class Program < ApplicationRecord
     'length' => { column: :length, converter: BaseConverter }
   }.freeze
 
-  validates :facility_code, :description, presence: true
+  validates :facility_code, presence: true
+  validates :description, presence: true
```

Comment:
do these need to be on a separate line?

Reply:
Yes there are quite a few like this, i'll fix all the models